composer.json

```diff
@@ -18,9 +18,11 @@
     "require": {
         "php": ">=5.5.9",
         "doctrine/common": "~2.4",
+        "fig/link-util": "^1.0",
```

Comment:
It's not `require`d by any of the packages in `replace`, so it should indeed remain in `require-dev`.